docs/source/maintainers_guidelines.rst

```diff
@@ -50,7 +50,6 @@ Examples of breaking changes include:
 Examples of non-breaking changes include:
 
 - Adding or removing an optional endpoint or field
```

Comment:
I am not sure what you intend here with optional endpoint or field.

Maybe would be better to says something like: 
- adding a feature to process a new sensor;
- adding a feature to create new DISDRODB  L1 and L2 products/fields;
- adding a feature providing new functionality to the DISDRODB API.

Reply:
Ok, I'll add this